modules/drivers/gnss/proto/config.proto

```diff
@@ -118,17 +125,18 @@ message Config {
   oneof device_config {
     NovatelConfig novatel_config = 7;
     UbloxConfig ublox_config = 8;
+    NewtonM2Config newtonm2_config = 9;
   }
 
   enum RtkSolutionType {
     RTK_RECEIVER_SOLUTION = 1;
     RTK_SOFTWARE_SOLUTION = 2;
   }
-  optional RtkSolutionType rtk_solution_type = 9;
-  optional ImuType imu_type = 10;
-  optional string proj4_text = 11;
-  optional TF tf = 12;
+  optional RtkSolutionType rtk_solution_type = 10;
+  optional ImuType imu_type = 11;
+  optional string proj4_text = 12;
+  optional TF tf = 13;
   // If given, the driver will send velocity info to novatel with command stream
-  optional string wheel_parameters = 13;
-  optional ZhdConfig zhd_config = 14;
+  optional string wheel_parameters = 14;
+  optional ZhdConfig zhd_config = 15;
```

Comment:
Please do not change the existing proto field index number. You may append new ones. Otherwise it will break the pre-recorded data processing.

Reply:
I have already modified the comment you left, please review again.